### PR TITLE
Fixed all the issues to do with multiple column sort. 

### PR DESCRIPTION
**Pull request by RustComet**
Needed to use POSTGRES methods to get it done. Hopefully this all goes smoothly

### CODE REVIEW

**Comment by RustComet** [comment]
At the moment I am forcing Array length to be 2.
Flexibility comes after operability.

File: lib/paginative/models/ordering_helpers.rb [hunk 1]
```diff
@@ -3,7 +3,12 @@ module OrderingHelpers
     extend ActiveSupport::Concern
 
     def sanitized_ordering(table_name, field, order)
-      "#{table_name}.#{sanitize_column(field)} #{sanitize_column_direction(order)}"
+      if field.is_a? Array
+        return raise "Wrong number of sorting fields. Expected 2, got #{field.length}. If you want to sort by a singular field please pass field argument as a string rather than an array." unless field.length == 2
+        "'#{table_name}.#{sanitize_column(field[0])} || #{table_name}.#{sanitize_column(field[1])} #{sanitize_column_direction(order)}'"
```
Comment (RustComet) [inline lib/paginative/models/ordering_helpers.rb]: Just realised I might not need the `'` marks in this line

**Comment by tspacek** [comment]
Could we get some README action to explain how to use it, and why it's needed?

**Comment by RustComet** [comment]
@tspacek Yeah I am writing that up now. Just wanted to get it all passing and confident that it was working

File: spec/dummy/app/models/test_model.rb [hunk 2]
```diff
@@ -2,5 +2,5 @@ class TestModel < ActiveRecord::Base
   include Paginative::ModelExtension
 
   reverse_geocoded_by :latitude, :longitude
-  after_validation :reverse_geocode          # auto-fetch coordinates
+  # after_validation :reverse_geocode          # auto-fetch coordinates
```
Comment (tspacek) [inline spec/dummy/app/models/test_model.rb]: Why is this commented out, could it be deleted entirely?

**Pull request merged**